OpenRA.Mods.Common/LoadScreens/SheetLoadScreen.cs

```diff
@@ -56,22 +56,16 @@ public override void Display()
 				sheet = null;
 			}
 
-			if (sheet == null && Info.ContainsKey("Image"))
+			if (sheet == null && Info.TryGetValue("Image", out var image))
 			{
-				var key = "Image";
-				density = 1;
-				if (dpiScale > 2 && Info.ContainsKey("Image3x"))
-				{
-					key = "Image3x";
+				if (dpiScale > 2 && Info.TryGetValue("Image3x", out image))
```

Comment:
Won't this reset `image` set from "Image" to null? We still need to use this!

Reply:
It will indeed. I was attempting to find an elegant way to write this but it looks like we'll need to be somewhat long-winded to be correct.

Reply:
Would this be a valid case to suppress the warning?

Reply:
Fixed, just needed some extra variables.